CMakeLists.txt

```diff
@@ -224,9 +224,15 @@ if(WITH_SPDK)
   if(NOT WITH_BLUESTORE)
     message(SEND_ERROR "Please enable WITH_BLUESTORE for using SPDK")
   endif()
-  include(BuildSPDK)
-  build_spdk()
-  set(HAVE_SPDK TRUE)
+  find_package(uuid REQUIRED)
```

Comment:
`Finduuid.cmake` is not included by cmake or ceph. how come this passes your local test?

Reply:
patch your pr， then patch mine